README.md

```diff
@@ -1,16 +1,24 @@
 # docker-cilkplus
 A small and simple dev environment for building and running C/C++ Cilk Plus applications with the GNU Compiler. This image was specifically built to avoid using the 4GB virtual machine image that is provided for the COMP 4009 Fall 2016 course at Carleton University.
 
+This was most recently tested for COMP 4009 A1 in Fall 2018.
```

Comment:
Nice! I like that you added this

Reply:
I mean it may change when I get my assignment mark back :stuck_out_tongue: but we'll see